### PR TITLE
set and get output helper methods

### PR DESCRIPTION
**Pull request by martinilevi**

### CODE REVIEW

File: log.go [hunk 1]
```diff
@@ -75,6 +75,15 @@ func NewLogger() *Logger {
 	}
 }
 
+
+// NewLoggerWithOutput to create a Logger to a different output than stdout.
```
Comment (jlorgal) [inline log.go]: I'd prefer to avoid a new constructor

File: log.go [hunk 2]
```diff
@@ -100,6 +109,16 @@ func (l *Logger) GetLevel() string {
 	return LogLevelNames[l.logLevel]
 }
 
+// SetOutput to set the log writer
```
Comment (jlorgal) [inline log.go]: Better to rename as SetWriter/GetWriter

**Review by jlorgal — CHANGES_REQUESTED** [review]

File: log.go (hunk 1 above)
Comment (martinilevi) [reply log.go]: ok

File: log.go (hunk 2 above)
Comment (martinilevi) [reply log.go]: ok

**Review by jlorgal — APPROVED** [review]

**Pull request merged**